types/nightwatch/index.d.ts

```diff
@@ -1493,6 +1492,77 @@ export function element(locator: string | ElementProperties | By | WebElement, o
 
 export type NightwatchTests = NightwatchTestFunctions | NightwatchTestHooks;
 
+export class DescribeInstance {
+    '[instance]': any;
+    '[attributes]': {};
+    '[client]': NightwatchClient;
+    name(): any;
+    tags(): any;
```

Comment:
When I've used tags in tests it normally takes a string array. Should we narrow it to that here?

Reply:
I am not sure, not used `tags` much.